components/autocomplete/README.md

```diff
@@ -0,0 +1,121 @@
+Autocomplete
+============
+
+TODO
```

Comment:
It can be very brief - usually, one line is enough :)

Reply:
Adding a one line description works for me. :) I was thinking I needed to document props for the `Autocomplete` component as well since the README didn't yet exist, but that doesn't have to be part of this PR.

Reply:
Follow up is fine